src/components/features/ProfileView.tsx

```diff
@@ -0,0 +1,56 @@
+import { useGetProfileInformationQuery } from "../../gql/graphql";
+import { formatDate } from "../../libs/utils";
+import ProfileSkeleton from "../ui/loadingSkeletons/ProfileSkeleton";
+import LinkIcon from "../../assets/icons/link-icon.svg?react";
+import LocationIcon from "../../assets/icons/location-icon.svg?react";
+import LinkedinIcon from "../../assets/icons/linkedin-icon.svg?react";
+import GithubIcon from "../../assets/icons/github.icon.svg?react";
+
+const ProfileView = () => {
+  const { data, loading } = useGetProfileInformationQuery();
+
+  if (loading) return <ProfileSkeleton />;
+  return (
+    <section className="flex w-full flex-col items-center gap-4">
+      <article className="flex h-auto w-full md:w-[400px] flex-col items-center gap-12 rounded-lg bg-color_neutral_4 p-4 text-color_neutral_1">
+        <div className="flex w-full h-[107px] items-center justify-center text-center relative text-body-xL font-bold bg-color_neutral_5 rounded-lg">
+          <img
+            alt={data?.profile.fullName}
+            className=" absolute top-12 left-5 w-[86px] rounded-full"
+            src={
+              data?.profile.avatar ||
+              "https://eu.ui-avatars.com/api/?name=HenryAgustin&size=250"
```

Comment:
I would suggest you to store this url as a constant, can be a global one if needed instead of passing it hardcoded here

so it could be something like `src={data?.profile.avatar || fallbackAvatarUrl}`